README.md

```diff
@@ -2,8 +2,83 @@
+Currently, we support only Unix-like operating systems like [Ubuntu](https://ubuntu.com). MacOs is still on our roadmap.  
+For windows systems you could use [wsl](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux) to set that up read more [here](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+
+### Dependencies
+One of the of goals of FileFighter is, that the client, that's you, only needs to met only one single dependency.  
+You need **Docker**.    
+Docker is a way to organise and run multiple applications. You can imagine it like a virtual machine (technical it`s a bit different), with a small file- and operating system within your machine.
+Sounds more scary than it actually is. Different Applications run in different containers, these only contain the necessary software to run the application.  
+The containers use the resources of the host machine, depending on the load.  
+Advantages are security and the possibility to easily shutdown and update the services. See more under [Usage](#Usage).
+
+#### Docker
+First check whether you have Docker already installed.  
+To do that run:  
+```shell script
+docker -v
+```
+If you see something like that (your version might be different):
+```shell script
+Docker version 19.03.13, build 4484c46d9f
+```
+You are good to go, and you can skip to [Usage](#Usage).
+
+To install **Docker** on Unix you can either use [snap](https://www.howtogeek.com/660193/how-to-work-with-snap-packages-on-linux/) or [apt](https://en.wikipedia.org/wiki/APT_(software)) as a package manager.  
+With snap its easier but of course it's not always possible to use snap.  
+
+##### Install with Snap
+To install Docker with [snap](https://www.howtogeek.com/660193/how-to-work-with-snap-packages-on-linux/) you can run:
+```shell script
+sudo snap install docker
+```
+##### Install with Apt
+Installing with [apt](https://en.wikipedia.org/wiki/APT_(software)) is a bit more difficult you can read [here](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-Docker-on-ubuntu-20-04) more about it.
 
 ## Usage
+After successfully installing Docker you can start using FileFighter.  
+This Repository contains one <!-- three --> important script:
+* The [initial start](./init_setup.sh) script starts downloads all the application and starts them in different containers. Run it with 
+```shell script
+./init_setup
+```
+After starting the script you should see something like the following:
+```shell script
+-------------------------< FileFighter >--------------------------
+|              Version 1.0 Last updated at 14.10.20              |
+|         Developed by Gimleux, Valentin, Open-Schnick.          |
+|       Development Blog: https://filefighter.github.io          |
+|  The code can be found at: https://www.github.com/filefighter  |
+--------------------< Started Initial Setup >---------------------
+```
+After the script succeeds you should be able to see the FileFighter Application.
+
```

Comment:
What do you need to do after installation? Login as admin and create users.

Reply:
> Login as admin and create users.

The question is how can we do that? Are we setting a default user as admin?

Reply:
Maybe we could use Nextcloud's workflow for orientation. You create an admin user while setup who can edit the settings as well as add more users to the cloud manually

Reply:
And how are we going to implement that?

Reply:
Deploying the nas including a default admin user admin:admin; password needs to be changed on first login?

Reply:
so a flag in the db for each use. Something like:
`"needsToChangePw": true`

Reply:
for example. Another option would be add some lines in the password validation process which check if the password is still the default password

Reply:
So we just set one user as default user and check it in the frontend if its the default one.